packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -493,10 +594,10 @@ class _DraggableScrollableSheetScrollPosition
     // Stop the ballistic animation if a new activity starts.
     // See: [beginActivity].
     _ballisticCancelCallback = ballisticController.stop;
-    double lastDelta = 0;
```

Comment:
There was some weird behavior here where this variable was named and initialized as if it were a delta, but was being used in `_tick` as an absolute position. Initializing it to the wrong value (0 as opposed to the current extent in pixels) wasn't causing any errant behavior because the error was being subtracted back out on L498, but I've fixed this anyway because this could introduce a confusing bug if L498 ever gets refactored.

Reply:
Nah this is fine.